# Written and placed in public domain by Jeffrey Walton.
# This patch fixes some issues with Mandoc. Also see
# https://git.alpinelinux.org/aports/tree/main/mandoc/mandoc_configure.patch
--- configure
+++ configure
@@ -41,7 +41,7 @@
 OSNAME=
 UTF8_LOCALE=
 
-CC=`printf "all:\\n\\t@echo \\\$(CC)\\n" | env -i make -sf -`
+CC=`printf "all:\\n\\t@echo \\\$(CC)\\n" | env -i PATH="${PATH}" make -sf -`
 CFLAGS=
 LDADD=
 LDFLAGS=
